app/src/main/java/com/github/featuredetectandroid/utils/PhotoAnalyzer.kt

```diff
@@ -42,6 +44,17 @@ class PhotoAnalyzer(private val imageViewModel: GrayscaleViewModel) : ImageAnaly
             width = height.also { height = width }
         }
 
+        featureDetector?.width = width
+        featureDetector?.height = height
```

Comment:
Let's not change it for every frame as it doesn't really change. Determine it somehow from the camera after it's set and save somewhere.

Reply:
fixed